estate/views/estate_property_views.xml

```diff
@@ -0,0 +1,146 @@
+<?xml version="1.0"?>
+<odoo>
+  <record id="estate_property_model_action" model="ir.actions.act_window">
+    <field name="name">Property</field>
+    <field name="res_model">estate.property</field>
+    <field name="view_mode">list,form</field>
+  </record>
+
+  <record id="estate_property_type_action" model="ir.actions.act_window">
+    <field name="name">Settings</field>
+    <field name="res_model">estate.property.type</field>
+    <field name="view_mode">list,form</field>
+  </record>
+
+  <record id="estate_property_taf_action" model="ir.actions.act_window">
```

Comment:
```suggestion
  <record id="estate_property_tag_action" model="ir.actions.act_window">
```
This is just a small typo, but it means that the `estate_property_tag_menu_action` menu item that references this action is not working